cmd/registry/metrics/metrics.go

```diff
@@ -0,0 +1,51 @@
+package metrics
+
+import (
+	"github.com/apigee/registry/rpc"
+)
+
+// ComputeMetrics will compute the metrics proto for a list of Classified Diffs.
+func ComputeMetrics(calculatedDiffs []*rpc.ClassifiedChanges) *rpc.Metrics {
+	stats := calculateStats(calculatedDiffs)
+	return caluclateMetrics(stats)
+}
+
+func calculateStats(calculatedDiffs []*rpc.ClassifiedChanges) *rpc.Stats {
+	totalBreakingChanges := 0
+	totalNonbreakingChanges := 0
+	for _, caluclatedDiff := range calculatedDiffs {
+
+		totalBreakingChanges = (totalBreakingChanges +
+			len(caluclatedDiff.BreakingChanges.Additions) +
+			len(caluclatedDiff.BreakingChanges.Deletions) +
+			len(caluclatedDiff.BreakingChanges.Modifications))
+
+		totalNonbreakingChanges = (totalNonbreakingChanges +
+			len(caluclatedDiff.NonBreakingChanges.Additions) +
+			len(caluclatedDiff.NonBreakingChanges.Deletions) +
+			len(caluclatedDiff.NonBreakingChanges.Modifications))
+
+		// Default Unknown Changes to Nonbreaking.
+		totalNonbreakingChanges = (totalNonbreakingChanges +
+			len(caluclatedDiff.UnknownChanges.Additions) +
+			len(caluclatedDiff.UnknownChanges.Deletions) +
+			len(caluclatedDiff.UnknownChanges.Modifications))
```

Comment:
Consider adding these to a separate counter, then combining the two counts when you create the stats message. That might make it clearer that we're treating those as the same in output, even though they're different as input.

Reply:
Done